### PR TITLE
partial care for empty lines in pyx in rings ; activate E302 in pyx

### PR DESCRIPTION
**Pull request by fchapoton**
this is fixing blank lines issues in pyx files in `rings` folder ; and activating the check for pycodestyle  E302 in all pyx files

### :memo: Checklist

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

File: src/sage/rings/polynomial/polynomial_number_field.pyx [hunk 1]
```diff
@@ -69,14 +69,15 @@ We can also construct polynomials over relative number fields::
 #  Distributed under the terms of the GNU General Public License (GPL)
 #  as published by the Free Software Foundation; either version 2 of
 #  the License, or (at your option) any later version.
-#                  http://www.gnu.org/licenses/
+#                  https://www.gnu.org/licenses/
 #*****************************************************************************
```
Comment (dcoudert) [inline src/sage/rings/polynomial/polynomial_number_field.pyx]: on the way:
```diff
- #*****************************************************************************
+ # ****************************************************************************
```

**Review by dcoudert — CHANGES_REQUESTED** [review]
otherwise LGTM.

**Comment by fchapoton** [comment]
merci pour les relectures !

**Review by dcoudert — APPROVED** [review]
LGTM.

**Pull request merged**